test/functional/ui/statuscolumn_spec.lua

```diff
@@ -520,4 +520,65 @@ describe('statuscolumn', function()
       :                                                    |
     ]])
   end)
+
+  it("has correct width when toggling '(relative)number'", function()
```

Comment:
nit: tests could be less verbose with:

        screen:try_resize(screen._width, 6)

Reply:
Well yeah, I think that holds for a lot of tests haha